option-parser.js

```diff
@@ -88,6 +88,14 @@ module.exports.fontGenerator =
 		)) ||
 	fontGenerator(enact.theme || 'moonstone');
 
+//  Customize @moon-accent LESS variable value
+module.exports.accent =
+	enact.accent &&
+	Object.assign(
+		{'moon-accent': enact.accent},
+		enact.theme && enact.theme !== 'moonstone' && {[enact.theme + '-accent']: enact.accent}
```

Comment:
This seems very specific and undocumented. Could you add detail as to what scenario the user is supposed to know about to prepare this to work? (Even if it's just for future-us, so we know what all this means.)